pkg/model/openstackmodel/servergroup.go

```diff
@@ -148,7 +159,7 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 			Tags: []string{
 				fmt.Sprintf("%s=%s", openstack.TagKopsInstanceGroup, groupName),
 				fmt.Sprintf("%s=%s", openstack.TagKopsName, portTagKopsName),
-				fmt.Sprintf("%s=%s", openstack.TagClusterName, b.ClusterName()),
+				fmt.Sprintf("%s=%s", openstack.TagClusterName, TruncateString(b.ClusterName(), 42)),
```

Comment:
perhaps we should add comment here why we are doing this

Reply:
error message: {“NeutronError”: {“type”: “InvalidInput”, “message”: “Invalid input for operation: ‘KubernetesCluster=cluster_name_of_more_than_42_characters’ exceeds maximum length of 60.“, “detail”: “”}}

Reply:
Added comment as suggested, sorry for the confusion